.github/workflows/codeql-codescan.yml

```
@@ -0,0 +1,44 @@
+# Please read the README.md file in this directory that defines what should 
+# be placed in this file
+#
+##############################################################################
+##############################################################################
+
+name: codeql codescan workflow
+
+on:
+  pull_request:
+    branches:
+      - '**'  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)